wcfsetup/install/files/lib/system/user/notification/UserNotificationHandler.class.php

```diff
@@ -98,7 +99,8 @@ public function fireEvent(
         IUserNotificationObject $notificationObject,
         array $recipientIDs,
         array $additionalData = [],
-        $baseObjectID = 0
+        $baseObjectID = 0,
+        $contentLanguageID = 0
```

Comment:
As this is a new parameter, it can be properly typed (likely `?int $contentLanguageID = null`), but it might be undesirable for consistency. I'll leave it open whether this should be done.